packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -138,93 +139,64 @@ abstract class MaterialLocalizations {
   }
 }
 
-/// Localized strings for the material widgets.
+class _MaterialLocalizationsDelegate extends LocalizationsDelegate<MaterialLocalizations> {
+  const _MaterialLocalizationsDelegate();
+
+  @override
+  Future<MaterialLocalizations> load(Locale locale) => DefaultMaterialLocalizations.load(locale);
+
+  @override
+  bool shouldReload(_MaterialLocalizationsDelegate old) => false;
+}
+
+/// US English strings for the material widgets.
+///
+/// See also:
+///
+///  * [GlobalMaterialLocalizations], which provides material localizations for
+///    many languages.
 class DefaultMaterialLocalizations implements MaterialLocalizations {
   /// Constructs an object that defines the material widgets' localized strings
-  /// for the given `locale`.
+  /// for US English (only).
   ///
   /// [LocalizationsDelegate] implementations typically call the static [load]
   /// function, rather than constructing this class directly.
-  DefaultMaterialLocalizations(this.locale)
-      : this._localeName = _computeLocaleName(locale) {
-    assert(locale != null);
-    if (localizations.containsKey(locale.languageCode))
-      _nameToValue.addAll(localizations[locale.languageCode]);
-    if (localizations.containsKey(_localeName))
-      _nameToValue.addAll(localizations[_localeName]);
-
-    if (intl.NumberFormat.localeExists(_localeName)) {
-      _decimalFormat = new intl.NumberFormat.decimalPattern(_localeName);
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00', _localeName);
-    } else if (intl.NumberFormat.localeExists(locale.languageCode)) {
-      _decimalFormat = new intl.NumberFormat.decimalPattern(locale.languageCode);
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00', locale.languageCode);
-    } else {
-      _decimalFormat = new intl.NumberFormat.decimalPattern();
-      _twoDigitZeroPaddedFormat = new intl.NumberFormat('00');
-    }
+  DefaultMaterialLocalizations(Locale locale) {
+    _decimalFormat = new intl.NumberFormat.decimalPattern('en_US');
```

Comment:
can we hard-code this so the intl dependency goes entirely?

Reply:
Yes, in fact I'd intended to do that but it looks like I forgot. Will fix.

Reply:
Yes. All of the intl dependencies, including the ones added by https://github.com/flutter/flutter/pull/12324, have been removed.